pkg/secrethub/audit.go

```diff
@@ -62,3 +62,11 @@ func (it *AuditEventIterator) Next() (api.Audit, error) {
 	}
 	return audit, nil
 }
+
+type AuditEventIterationOption func(*AuditEventIterator) error
+
+func OnlySubjectTypes(subjectTypes api.AuditSubjectTypeList) AuditEventIterationOption {
+	return func(it *AuditEventIterator) error {
+		return http.OnlySubjectTypes(subjectTypes)(it.paginator)
+	}
+}
```

Comment:
Do I correctly understand that this means that we have to duplicate any option in both the `http` and the `secrethub` package?

Reply:
Yes, options on the paginator are passed to the `http` package (similarly as we do for client options).

Options on the iterator are of course not passed to the `http` package.